README.md

```diff
@@ -1,15 +1,142 @@
+# Description
 
-### Code of Conduct
+This boilerplate project is a project set up that can be easily altered to create new projects. The user is able to use in the original project, its foundation, and its structure to set up a new one without changing the original.
 
-Code of conduct guidelines [here.](CODE_OF_CONDUCT.md)
+This project is based on Angular CLI on version 14.0.0.
 
-### License
+# Prerequisite
 
-[MIT](LICENSE)
+Run `npm install -g @angular/cli` for Angular CLI & NPM installed
+Run `npm i @angular/material` for Angular material
 
-<!-- docs-index-end -->
+# Code scaffolding
+
+Run ng generate component component-name to generate a new component. You can also use ng generate directive|pipe|service|class|guard|interface|enum|module.
+
+## Project Features
+
+Angular Boilerplate is a starter template for Angular applications that provides various features and components to help developers get started with their projects quickly. Here are some of the features and components that the boilerplate provides:
+
+# Auth Module:
+
+This Module provides the following features:
+
+## Login Component:
+
+- login component allows users to authenticate themselves by entering their credentials, such as a username and password. The login component typically includes a form with input fields for the user's credentials, as well as a submit button to initiate the login process.
+
+- login component communicates with a backend server to verify the user's credentials and grant access to the application if the user is authorized. Upon successful authentication, the login component typically stores the user's session information, such as a token or cookie, to allow the user to access protected resources without having to log in again.
+
+## Auth Component:
+
+Auth component handles the authentication and authorization of users. It is responsible for managing user sessions, verifying user credentials, and granting access to protected resources based on the user's role and permissions.
+
+The auth component typically includes a login as well as a registration form for new users to create an account. The component may also handle password reset functionality and provide options for users to manage their accounts.
+
+# Core Component:
+
+## API:
+
+- Adapters:We are using 3 adapters in this microservice named "anyAdapter,countAdapter,nameAdapter".We use these adapters for manipulating the data, as we get the data from the api.
```

Comment:
Microservice?
Also fix the sentence formatting like: space after the colon and dot.

Reply:
Done